### DISCUSSION TITLE
How to load faster a .dxf file with ezdxf.readfile()

### DISCUSSION THREAD

**Discussion by Fra6008** [mozman/ezdxf#905] [Issues]
Hi guys, first time here :) 
Do you have any suggestion to load in a faster way a 100MB dxf with more than 30 Layers? 
Can I made a lighter dxf removing some layers without loading the whole file? 

Sorry if its the wrong place, thanks in advance for your answers

**Comment by mozman** [comment]
You are processing huge CAD files in a very verbose file format with a slow (but beautiful) scripting language.

The loading stage is written in pure Python  and the function `ezdxf.readfile()` is the fastest way to load DXF files and always loads the whole file, there is no possibility to skip content.

Try [PyPy](https://www.pypy.org/) if this Python implementation is an option for you. It can speed things up when startup time is a minor factor in your execution time.
